client.go

```diff
@@ -332,7 +331,7 @@ var (
 	// ErrNotFound is returned when a query by ID does not match any existing
 	// rows. For example, attempting to cancel a job that doesn't exist will
 	// return this error.
-	ErrNotFound = errors.New("not found")
+	ErrNotFound = rivertype.ErrNotFound
```

Comment:
Yeah, `errors.Is` still works because it's a type alias (with `=`).

I would've slightly preferred to not have it at the top-level `river` and only be in `rivertype`, but added this alias for backwards compatibility (although this is a change that _maybe_ we could get away with). It needs to be in `rivertype` so lower level packages can reference it.